shuup_tests/front/test_checkout.py

```diff
@@ -72,3 +77,102 @@ def test_address_phase_anonymous_user(rf):
     resp = view_func(request)
     assert 'company' in resp.context_data['form'].form_defs
 
+
+# BROWSER TESTS BELOW
```

Comment:
I'm not 100% sure I want to divide browser tests into their own files. I was thinking about this, but I think we need to make it later IF the tests are long. This test here is exceptionally long as it proves the whole checkout process.